### ISSUE TITLE
Wrong name of Ellipsis icon at website

### ISSUE THREAD

**Issue by caducoder** [lucide-icons/lucide#2104]
### Package

- [ ] lucide
- [ ] lucide-angular
- [ ] lucide-flutter
- [ ] lucide-preact
- [X] lucide-react
- [ ] lucide-react-native
- [ ] lucide-solid
- [ ] lucide-svelte
- [ ] lucide-vue
- [ ] lucide-vue-next
- [ ] Figma plugin
- [ ] source/main
- [ ] other/not relevant

### Version

0.343.0

### Can you reproduce this in the latest version?

- [X] Yes
- [ ] No

### Browser

- [ ] Chrome/Chromium
- [X] Firefox
- [ ] Safari
- [ ] Edge
- [ ] iOS Safari
- [ ] Opera
- [ ] Other/not relevant

### Operating system

- [X] Windows
- [ ] Linux
- [ ] macOS
- [ ] Other/not relevant

### Description

When I try to use the Ellipsis icon, I receive an error telling that the Ellipsis import not found:
![image](https://github.com/lucide-icons/lucide/assets/73801000/3ef63c38-94fb-45c7-808e-12d241542bca)

### Steps to reproduce

1. Import `Ellipsis` icon. 
2. Add to a React component/view 
3. Run the react app 
4. Notice the error on the console: "Uncaught SyntaxError: import not found: Ellipsis"

### Checklist

- [X] I have searched if someone has submitted a similar issue before and there weren't any. (Please make sure to also search closed issues, as this issue might already have been resolved.)

**Comment by South-Paw** [comment]
😂 also just hit this

I updated my `lucide-react` version to latest (0.341.0 -> 0.373.0), ran `npm i` and checked the package exports again - it seems like they disappeared from a release but after updating and re-installing they have now returned to the exports 🥳

**Comment by chengfengfengwang** [comment]
I am using 0.316.0，and I can't find Ellipsis

**Comment by chengfengfengwang** [comment]
I find that in v0.316.0, its name is MoreHorizontalIcon

**Comment by karsa-mistmere** [comment]
Duplicate of #1996 

TLDR: Lucide.dev is in sync with the latest version, please upgrade your packages.
